src/js/components/account/visualizations/sankey/SankeyVisualizationHorizontal.jsx

```diff
@@ -88,7 +88,7 @@ export default class SankeyVisualizationHorizontal extends React.Component {
         this.generateChart(this.props);
     }
 
-    componentWillReceiveProps(nextProps) {
+    componentDidUpdate(nextProps) {
         this.generateChart(nextProps);
```

Comment:
nextProps here is actually the components previous props. So doing this would actually invoke generateChart with stale props. 

https://reactjs.org/docs/react-component.html